src/content-helper/editor-sidebar/excerpt-suggestions/class-excerpt-suggestions.php

```diff
@@ -42,29 +42,28 @@ public function __construct( Parsely $parsely ) {
 	 * @return string The filter name.
 	 */
 	public static function get_feature_filter_name(): string {
-		return self::get_global_filter_name() . '_excerpt_generator';
+		return self::get_global_filter_name() . '_excerpt_suggestions';
```

Comment:
I think this will break existing usages of the `wp_parsely_enable_content_helper_excerpt_generator` filter. A strategy here would be to deprecate the old filter name, but still support if for some time. We should do the same if we've changed other filter names.

Reply:
Good call!

Reply:
I did addressed this on [b8e3978](https://github.com/Parsely/wp-parsely/pull/2776/commits/b8e3978b948c87578da32a098db3e4a6aacfe9bb). I ended up reverting back to the `excerpt_generator`, and it should send a deprecation notice (if WP_DEBUG is enabled), using `apply_filters_deprecated` and suggesting `wp_parsely_current_user_can_use_pch_feature`.

This required some tweaking since the `wp_parsely_enable_content_helper_excerpt_generator` was not even working properly, as now the excerpt generator does not have it's own JS and CSS files, so we can't simply not load it. So I hooked up that filter with the new permission feature of the plugin, so if `wp_parsely_enable_content_helper_excerpt_generator` is set and returning false, is as if the feature was disabled.